webapp/components/DeleteData/DeleteData.vue

```diff
@@ -42,8 +54,9 @@
 </template>
 
 <script>
-import { mapGetters, mapActions } from 'vuex'
+import { mapActions, mapGetters } from 'vuex'
 import gql from 'graphql-tag'
+import { currentUserCountQuery } from '~/graphql/User'
 
 export default {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-05-27T11:01:36Z" title="Wednesday, May 27th 2020, 1:01:36 pm +02:00">May 27, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -52,8 +61,19 @@ export default {
       deleteContributions: false,
       deleteComments: false,
       enableDeletionValue: null,
+      currentUserCount: [],
```


This makes sense here to have this in a separate variable in my eyes.

I would name it with plural `currentUserCounts`.
And this is initial no array. It’s an object:
`currentUserCounts: {},`

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-07-30T11:40:30Z" title="Thursday, July 30th 2020, 1:40:30 pm +02:00">Jul 30, 2020</time>_

---


ok